main.cpp

```diff
@@ -0,0 +1,53 @@
+#include "Main.h"
+
+//constructors
+Unit::Unit(){
+	std::string name;
+	int hp, damage;
+	std::cin >> name >> hp >> damage;
+	Unit::setName(name);
+	Unit::setHp(hp);
+	Unit::setDamage(damage);
+}

```

Comment:
Ne legyen input bekérés egy "backend" működést előállító osztályból.
A feladatot a kiírásnak megfelelően kell megoldani, azaz command arguments-ből kapja a paramétereket. (Ahogy az a példában is látható.)